### PR TITLE
made counter

### PR DESCRIPTION
**Pull request by ghost**
image

### CODE REVIEW

**Comment by Rudenkopolina** [comment]
remove please unnecessary files.

File: counter/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Counter</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="alert.js"></script>
```
Comment (Rudenkopolina) [inline counter/index.html]: rename to correct name

File: counter/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+  let count = 0;
+  const counter = {
+    next: () => {
+      var next = document.getElementById("counter");
```
Comment (Rudenkopolina) [inline counter/index.js]: please use ```let``` instead ```var```

**Review by Rudenkopolina — CHANGES_REQUESTED** [review]

File: counter/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  </head>
+  <body>
+    <div class="container">
+      <div class="title">
+        Counter
+      </div>
+      <div class="buttons">
+         <div class="button-add">
+          <button type="button" class="button" onClick="counter.next()">+1</button>
```
Comment (IvanZharnov) [inline counter/index.html]: `type="button"` is not required

File: counter/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+         </div>
+         <div class="button-reset">
+           <button type="button" class="button" onclick="counter.reset()">Reset</button>
```
Comment (IvanZharnov) [inline counter/index.html]: `type="button"` is not required

File: counter/index.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+         </div>
+         <div class="subtract">
+           <button type="button" class="button" onclick="counter.previous()">-1</button>
```
Comment (IvanZharnov) [inline counter/index.html]: `type="button"` is not required

**Review by IvanZharnov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**